datadog/metric_test.go

```diff
@@ -26,30 +26,24 @@ import (
 	"testing"
 	"time"
 
-	"github.com/google/ts-bridge/mocks"
-
 	"github.com/google/ts-bridge/datastore"
+	"github.com/google/ts-bridge/mocks"
 
 	"github.com/golang/mock/gomock"
 	"github.com/golang/protobuf/proto"
 	ddapi "github.com/zorkian/go-datadog-api"
-	"google.golang.org/appengine/aetest"
 	metricpb "google.golang.org/genproto/googleapis/api/metric"
 	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
 )
 
-var testCtx context.Context
-
 func TestMain(m *testing.M) {
-	var done func()
-	var err error
-	testCtx, done, err = aetest.NewContext()
-	if err != nil {
-		panic(err)
-	}
-
+	ctx, cancel := context.WithCancel(context.Background())
+	// Save the emulator's quit channel
```

Comment:
Go style says that comments should be full sentences, starting with a capital and ending with a `.`

Reply:
Done.